src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -28,6 +28,14 @@ namespace winrt::TerminalApp::implementation
         Initialized = 2
     };
 
+    enum ScrollDirection : int
+    {
+        // Hack alert, we are going to use numeric values to compute the actual scroll delta,
```

Comment:
I appreciate the call out, but some days I feel like this entire product is just one giant hack alert.

Reply:
I am new here - so checking the limits 😄 . I will fix this.

Reply:
Removed the hack. Sorry.

Reply:
HAH! This isn't a moment you need to apologize for. Michael is saying that we've got enough hacks that are bigger than this that we don't need to call this one out, and instead just do to it :smile: